docs/1-Using-Fleet/3-REST-API.md

```diff
@@ -17,6 +17,16 @@
   - [Create a user account with an invitation](#create-a-user-account-with-an-invitation)
   - [Create a user account without an invitation](#create-a-user-account-without-an-invitation)
   - [Get user information](#get-user-information)
+- [Fleet web application](#fleet-web-application)
```

Comment:
What do you think of this? 

```suggestion
- [Fleet configuration](#fleet-configuration)
```

Reply:
"Configuration" makes more sense to me. The data changed using these endpoints affects the use of not only the Fleet app but also the use of the API and fleetctl.